.github/pull_request_template.md

```diff
@@ -6,6 +6,10 @@ Closes [insert issue #]
 ## Solution
 <!-- How did you solve the problem? -->
 
+**Breaking Changes** 
+<!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->
+- Details ...
```

Comment:
Maybe add a checkbox also for ease of use (like in many open source issue templates)

- [ ] Yes
- [ ] No

Reply:
i think add numbers bah so it's clear cos we want to know the breaking change(s) ideally

Reply:
i mean, yes/no is just a bird's eye view, the details part is still there haha

Reply:
oh i thought the PR author will delete the section if no breaking changes HAHAHA ok will change!!